keyboards/planck/keymaps/kifinnsson/keymap.c

```diff
@@ -0,0 +1,584 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+bool     is_lgui_active = false;
+uint16_t lgui_timer     = 0;
+
+
+enum planck_layers { _COLEMAK, _LOWER, _RAISE, _EXTEND, _MACRO, _ADJUST };
+
+enum planck_keycodes {
+    COLEMAK = SAFE_RANGE,
+    KI_BSPC,
```

Comment:
You only have colemak as a layout, you can remove the keycode. 

```suggestion
    KI_BSPC = SAFE_RANGE,
```